vic/vic_run/include/vic_def.h

```diff
@@ -60,7 +60,7 @@
 #define MIN_VEGCOVER    0.0001 /**< Minimum allowable vegcover fraction */
 
 /***** Define minimum and maximum values for model timesteps *****/
-#define MIN_SUBDAILY_STEPS_PER_DAY  4
+#define MIN_SUBDAILY_STEPS_PER_DAY  1
```

Comment:
this we do not want.  Because we have removed `mtclim` in the image driver, you may run into a bunch of problems here.